components/calendar/Header.tsx

```diff
@@ -147,7 +147,7 @@ export interface CalendarHeaderProps<DateType> {
   locale: Locale;
   mode: CalendarMode;
   fullscreen: boolean;
-  onChange: (date: DateType) => void;
+  onChange: (date: DateType, selectType: CalendarSelectable) => void;
```

Comment:
```suggestion
  onChange: (date: DateType, selectInfo: {
    triggerType: CalendarSelectable,
  }) => void;
```

1. 放一个对象里，避免以后还有新的参数要加。
2. 改成 triggerType，日期更新不一定是用户选择行为。